### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by codenirvana**
- Update content ordering
- Add HTML reporter instructions
- Add contributing section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,91 +3,107 @@ _Supercharge your API workflow<br/>Modern software is built on APIs. Postman hel
...
-$ newman run https://www.getpostman.com/collections/631643-f695cab7-6878-eb55-7943-ad88e1ccfd65-JsLv;
+$ newman run https://www.getpostman.com/collections/631643-f695cab7-6878-eb55-7943-ad88e1ccfd65-JsLv
 ```
 
 For the complete list of options, refer the [Commandline Options](#commandline-options) section below.
 
 ![terminal-demo](https://raw.githubusercontent.com/postmanlabs/postmanlabs.github.io/develop/global-artefacts/newman-terminal.gif)
 
-### Using Newman as a NodeJS module
+### Using Newman Library
```
Comment (kunagpal) [inline README.md]: `as a `

File: README.md [hunk 2]
```diff
@@ -181,31 +195,66 @@ newman.run({
   Allows the specification of global variables via the command line, in a key=value format. Multiple CLI global variables
   can be added by using `--global-var` multiple times, like so: `--global-var "foo=bar" --global-var "alpha=beta"`.
 
-#### Configuring Reporters
+### `newman [options]`
```
Comment (kunagpal) [inline README.md]: Shift this above `newman run ...`

File: README.md [hunk 3]
```diff
@@ -496,31 +497,53 @@ In this collection, `sample-file.txt` should be present in the current working d
 }
 ```
 
-```terminal
+```console
```
Comment (kunagpal) [inline README.md]: ?

Comment (codenirvana) [reply README.md]: Better shell highlighting,

console:
```console
foo@bar:~$ whoami
foo

$ ls
file-upload.postman_collection.json  sample-file.txt

$ newman run file-upload.postman_collection.json
```
terminal:
```terminal
foo@bar:~$ whoami
foo

$ ls
file-upload.postman_collection.json  sample-file.txt

$ newman run file-upload.postman_collection.json
```

**Pull request merged**